src/main/java/com/example/shorturl/model/user/UserEntity.java

```diff
@@ -0,0 +1,54 @@
+package com.example.shorturl.model.user;
+
+
+import com.example.shorturl.model.roles.RoleEntity;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.CascadeType;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.JoinTable;
+import javax.persistence.ManyToMany;
+import javax.persistence.Table;
+import java.util.ArrayList;
+import java.util.List;
+
+@Data
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+@Entity
+@Table(name = "users")
+public class UserEntity {
+
+    @Id
+    @Column(name = "user_id")
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private int userId;
+    private String userName;
+    private String password;
+    private String email;
+    private String firstName;
+    private String lastName;
+
+    @ManyToMany(cascade = CascadeType.ALL, fetch = FetchType.EAGER)
```

Comment:
Why EAGER?

Reply:
This is how it was used in the tutorial, let me look for information because I think, this can be improved

Reply:
Changed to LAZY